zmon-actuator/src/test/java/org/zalando/zmon/actuator/ZmonMetricsFilterTest.java

```diff
@@ -60,31 +49,32 @@
     private int port;
 
     @Autowired
-    private MetricRegistry metricRegistry;
+    private MeterRegistry meterRegistry;
 
     private RestTemplate restTemplate;
 
     private final Random random = new Random(System.currentTimeMillis());
 
     @Before
     public void setUp() {
-        ConsoleReporter reporter = ConsoleReporter.forRegistry(metricRegistry).convertRatesTo(TimeUnit.SECONDS)
-                                                  .convertDurationsTo(TimeUnit.MILLISECONDS).build();
-        reporter.start(2, TimeUnit.SECONDS);
-
+//        Reporter reporter = Reporter.forRegistry(meterRegistry).convertRatesTo(TimeUnit.SECONDS)
```

Comment:
TODO: find a replacement for the console logger or remove the commented code

Reply:
I dropped the code.